workshops/test/test_person.py

```diff
@@ -947,9 +951,12 @@ def test_all_requirements_satisfied(self):
                          set())
 
     def test_some_requirements_are_fulfilled(self):
+        # Homework was accepted, the second time.
```

Comment:
> Is there still a test somewhere for "a single passed homework object was created"?

This case is covered in [`test_all_requirements_satisfied`](https://github.com/swcarpentry/amy/pull/1017/files/399d4d242e52d35e32c81200274f20cc714b50b3#diff-ab64c8508f4039f3a209a9ca973a9a24R889).

> Also, how are you testing that everything worked - just by the fact that there's no exception thrown?

No, we use `assertEqual` [here](https://github.com/swcarpentry/amy/pull/1017/files/399d4d242e52d35e32c81200274f20cc714b50b3#diff-ab64c8508f4039f3a209a9ca973a9a24R921). I'm not sure if I understood your question.